cli/cmd/plugin/ui/web/tanzu-ui/node-server/src/conf/appConfig.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+//
+// Web Application Configuration
+//
+const _ = require('lodash');
+const assert = require('assert');
+const path = require('path');
+const paths = require('./paths');
+const libUtil = require(paths.src.util);
+
+//
+// Module
+//
+
+// @see #def.port
+let __port;
+// @see #def.userDataPath
+let __userDataPath;
+// @see #def.logPath
+let __logPath;
+// @see #def.clientPath
+let __clientPath;
+// @see #def.logLevel
+let __logLevel;
+// @see #def.apiEndpoint
+let __apiEndpoint;
+// @see #def.sampleJsonDirectory
+let __sampleJsonDirectory;
+// @see #def.__serverErrorState
+let __serverErrorState;
+
+// [internal]
+// @const {Number} Default app port.
+const DEFAULT_PORT = 8008;
+// @const {String} Default log level
+const DEFAULT_LOG_LEVEL = 'info';
+
+// {Object.<namespace:String,Object>} module definition
+let def = {
```

Comment:
`def` object reference is never changed, so please use `const` instead of `let`.